src/client.ts

```diff
@@ -28,12 +28,43 @@ export const defaultConfig = (): NerisApiConfig => {
   }
 };
 
-export const createNerisClient = (config: NerisApiConfig = defaultConfig()) => {
+export type NerisClient = Client<paths> & {
+  fetchGeometry: (url: URL) => Promise<components['schemas']['GeoJsonMultiPolygon']>;
+};
```

Comment:
`fetchGeometry` is an extension to call the `pygeoapi` with credentials. Meant to be used to hydrate the geometry from a url.  

[There is an example here](https://github.com/ulfsri/neris-nodejs-client/pull/9/files#diff-007b0b27e23b789227e3df829bd08bd7ceb475e90bb7effde5a32cda0664fd7aR135)